src/components/topbar/wallet/wallet.tsx

```diff
@@ -85,135 +108,6 @@ function getErrorMessage(error: Error) {
   }
 }
 
-function ChainId() {
```

Comment:
Why was this removed? This logic is necessary but please explain the reason behind it.